docker-compose.yml

```diff
@@ -97,14 +97,16 @@ services:
       - ./serve-web/localstack-data:/tmp/localstack/data
 
   postgres:
-    image: postgres:9.6-alpine
```

Comment:
Do we want to be using latest for this? Not sure what we use in prod but we probably want to pin to it.